lms/static/js/learner_dashboard/program_list_factory.js

```diff
@@ -19,6 +28,13 @@
                 el: '.sidebar',
                 context: options
             }).render();
+
+            new CollectionListView({
+                el: '.certificate',
+                childView: CertificateView,
+                collection: new CertificateCollection(options.certificatesData)
+            }).render();
```

Comment:
Instead of just having a collection list view I would prefer a self-contained certificate view that adds the code in the sidebar.underscore;
```
<p class="title"><%- gettext('XSeries Program Certificates') %>:</p>
<div class="certificate-image"></div>
<div class="certificate"></div>
```
And then runs through the CollectionsListView and adds the certificate links. That was the view will be easily reusable. The current implementation of the sidebar is not ideal, the explore more xseries view should really be self-contained too. After this PR has merged I will update the explore more xseries view to follow this convention.